test/abstract/FlowBasicTest.sol

```diff
@@ -23,12 +23,12 @@ abstract contract FlowBasicTest is FlowUtilsAbstractTest, InterpreterMockTest {
     using Address for address;
 
     CloneFactory internal immutable iCloneFactory;
-    IFlowV5 internal immutable iFlowImplementation;
+    address internal flowImplementation;
```

Comment:
there's no reason to have a mutable storage value for this, if it's going to be for building contracts the builder should just return addresses for local scope